<launch>
    <!--<param name ="/use_sim_time" value="true"/>-->

    <node pkg="ibeo_scala" type="scala_rviz_display" name="scala_rviz_display" output="screen" />

    <param name="robot_description" textfile="$(find ibeo_scala)/urdf/scala_tiggo.urdf" />

    <!--<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />-->

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ibeo_scala)/scala_display.rviz" required="false" />

</launch>
